source/_docs/ecosystem/certificates/tls_domain_certificate.markdown

```diff
@@ -45,7 +45,8 @@ Press Enter to Continue
 
 ### Make mydomain.com point to your Home Assistant instance
 
-If your router uses DNSMasq (for example DDWRT) add the following line to DNSMasq options:
+* Deploy IP addres to A field using your domain registrar. Remember, it may take a few minutes for the setting to take effect. It can work with ZeroTier address to.
```

Comment:
That is actually a really bad security practice.
I would recommend against having this in our default documentation.

Reply:
Why is this bad security practice?

Reply:
oh wait, I might have misread this.

What IP address? Your external? The one of your device (internal)?

Reply:
Internal LAN IP address. Actualy I use fake ZeroTier address. Btw thanks for addon. And what is security issue? I set it up only today, but I am curious if I does not miss something.

Reply:
You should not publish information about your internal LAN structure publicly. Those types of things should be done using a DNS split with a local resolver.

Reply:
I am behind NAT, and i use hosts record on my openWRT router for local domain name translate as is writen in documentation right now. But what can be local resolver when there is remote access trough ZeroTier?

Reply:
In my opinion it is safe. Because even if IP address of the local machine is public. It is not clear which network it is on. But I am no security expert.